backend/btrixcloud/operator/crawls.py

```diff
@@ -755,8 +761,8 @@ async def sync_crawl_state(
                     )
 
                 if not crawler_running and redis:
-                    # if crawler running, but no redis, stop redis instance until crawler
-                    # is running
+                    # if crawler is not running, but redis is, stop redis instance until crawler
+                    # is running, and REDIS_TTL has elapsed, stop redis
```

Comment:
Not quite sure what this means, could use some clarification/rewording - maybe the doubled "stop redis" is the source of confusion?

Reply:
yeah, this probably could've been a separate PR, basically resets the lastActiveTime to ensure redis isn't immediately shutdown when a pod is (eg. due to pod being restarted for mem resizing). the lastActiveTime wasn't being updated so TTL wasn't set correctly.